components/compliance-service/ingest/pipeline/processor/control_worker.go

```diff
@@ -73,10 +73,12 @@ func (s *ControlWorkflow) OnStart(w cereal.WorkflowInstance,
 
 	workflowPayload.ReportUuid = workflowParams.ReportUuid
```

Comment:
We can merge the codes in L62 and below ones:
```suggestion
	workflowPayload = ControlWorkflowPayload{ ReportUuid: workflowParams.ReportUuid, RetriesLeft:  workflowParams.Retries, EndTime: workflowParams.Retries, Status: RunningStatus}
```

Reply:
done